src/escalus_auth.erl

```diff
@@ -161,23 +161,23 @@ scram_sha1_response(Conn, GS2Headers, ClientFirstMessageBare, Props) ->
     Nonce = get_property(<<"r">>, ChallengeData),
     Iteration = binary_to_integer(get_property(<<"i">>, ChallengeData)),
     Salt = base64:decode(get_property(<<"s">>, ChallengeData)),
-    SaltedPassword = scram:salted_password(Password, Salt, Iteration),
-    ClientKey = scram:client_key(SaltedPassword),
-    StoredKey = scram:stored_key(ClientKey),
+    SaltedPassword = escalus_scram:salted_password(Password, Salt, Iteration),
+    ClientKey = escalus_scram:client_key(SaltedPassword),
+    StoredKey = escalus_scram:stored_key(ClientKey),
     GS2Headers64 = base64:encode(GS2Headers),
     ClientFinalMessageWithoutProof = <<"c=",GS2Headers64/binary,",r=", Nonce/binary>>,
```

Comment:
Line 168 is too long:     ClientFinalMessageWithoutProof = <<"c=",GS2Headers64/binary,",r=", Nonce/binary>>,.